practice/src/index.html

```diff
@@ -142,5 +142,65 @@ <h4 class="secondary-title">Popular Trip Packages</h4>
       </div>
     </section>
   </main>
+
+  <footer class="footer-section">
+    <!-- Footer section -->
+    <div class="container footer-main">
+      <div class="footer-row">
+        <div class="footer-col">
+          <a href="javascrip:void(0)" class="footer-logo">
+            <img src="../src/assets/images/footer/logo-footer.svg" alt="Logo large for footer">
+            <span class="footer-logo-name">DamiumTrip's</span>
+          </a>
+          <p class="text footer-copyright footer-text">Copyright © 2023. All Rights Reserved.</p>
```

Comment:
Should use HTLM symbol https://www.w3schools.com/html/html_symbols.asp instead of  ©